manifests/apache.pp

```diff
@@ -60,6 +60,17 @@
     content => template('katello/pulp-apache-ssl.conf.erb'),
   }
 
+  if $katello_devel::pulp3 {
+    class { 'pulpcore':
+      manage_apache => false,
+    }
+
+    concat::fragment { 'katello-pulpcore':
+      target  => '05-katello.conf',
```

Comment:
This doesn't exist. You can also use a simpler way: add it to $proxy_pass_https above.